ircd/s_user.c

```diff
@@ -972,6 +972,33 @@ report_and_set_user_flags(struct Client *source_p, struct ConfItem *aconf)
 	}
 }
 
+void
+report_priv_change(struct Client *client, struct PrivilegeSet *old, struct PrivilegeSet *new)
+{
+	const struct PrivilegeSet *added, *removed, *unchanged;
+	if (old == NULL && new == NULL)
```

Comment:
Should this logic be in s_conf.c and then that file can call into this function with the right old and new? Do we expect this behavior anywhere else?